frontend/test/pytest/test_template.py

```diff
@@ -599,8 +901,273 @@ def qnn(weights, inputs):
     assert np.allclose(expected, observed)
 
 
-# Hilbert Schmidt templates take a quantum tape as a parameter.
-# Therefore unsuitable for JIT compilation
```

Comment:
What made HilbertSchmidt to be JIT compatible? Just changing the way it is called inside a qjit workflow?

Reply:
Something about the quantum tape as input breaks QJIT even though it works with JIT.